docs/built-in-functions.rst

```diff
@@ -143,6 +146,15 @@ Chain Interaction
         def foo(_target: address) -> Bytes[32]:
             response: Bytes[32] = raw_call(_target, 0xa9059cbb, max_outsize=32, value=msg.value)
             return response
+            
+        @external
+        @payable
+        def bar(_target: address) -> Bytes[32]:
+            success: bool = True
```

Comment:
I would set this to `False` as a general practice

Reply:
```suggestion
            success: bool = False
```